scst-store/logs.hbs.md

```diff
@@ -34,6 +34,10 @@ extensibility.
 
 If no verbosity level is specified when the Store is installed, the level is set to `default`.
 
+### <a id='slow-sql'></a> Slow SQL
+
+Slow sql statement will be logged only when verbosity level is set to trace and sql query took more than 200 milli seconds to execute. You can change this default by setting `db_slow_threshold_ms` value in values.yaml file and redeploy metadata store.
```

Comment:
Can we highlight the word trace here? It's a key value that users can set it to right? 
Also does capitalization matter?

Reply:
Made required change but can't do Capital since it is not correct verbosity Level